typpete/inference_runner.py

```diff
@@ -44,19 +44,23 @@ def configure_inference(flags):
             print("Invalid flag {}. Ignoring.".format(flag_name))
     return class_type_params, func_type_params
 
-def run_inference(file_path=None):
-    if not file_path:
-        try:
-            file_path = sys.argv[1]
-        except:
+
+def run_inference(file_name=None, base_folder=None):
+    if not file_name:
+        if len(sys.argv) >= 2:
+            file_name = sys.argv[1]
+            if len(sys.argv) >= 3:
+                base_folder = sys.argv[2]
+        else:
             print("Please specify the python file")
             return
-
     start_time = time.time()
     class_type_params, func_type_params = configure_inference([flag for flag in sys.argv[2:] if flag.startswith("--")])
 
-    base_folder = '/'.join(file_path.split('/')[:-1])
-    file_name = file_path.split('/')[-1]
+    if not base_folder:
+        print("WARNING: No root directory specified. Using the current working directory")
```

Comment:
Are you sure we need a warning like this every time? I think it would be better if the default output (the one above that currently only says "Please specify the python file") would describe the supported command line options.

Reply:
This warning has nothing to do with not providing the input python file right? You can still run the inference without specifying the root directory, but you cannot do so without specifying the input file. I think we can have a help flag (like `-h`) that displays all the command line options, do you agree?

Reply:
Yeah, these were really two comments. 1. I don't really think we need the warning (at least not with the word WARNING, that sounds very dramatic and as if something is probably wrong), and 2. yes, we should have the tool explain the options. I agree, a -h flag would be good.

Reply:
Done